tests/components/kodi/test_init.py

```diff
@@ -9,11 +7,7 @@
 
 async def test_unload_entry(hass):
     """Test successful unload of entry."""
-    with patch(
```

Comment:
is this not needed??

Reply:
The test is successful without it. Furthermore `homeassistant.components.kodi.media_player.async_setup_entry` doesn't have a return value and I could simplify `init_integration` because of the connection extraction.